apps/cyberstorm-storybook/public/css/globals.css

```diff
@@ -97,6 +97,7 @@
   --color-border: var(--color-default-purple--6);
   --color-border-2: var(--color-default-purple--7);
   --color-border-3: var(--color-default-purple--8);
+  --color-shadow-1: #000000a6;
```

Comment:
Not a blocker fot this PR. This and others like this aren't probably from this PR, but to note it down, using `rgb` for colors with alpha channel value is preferred. In fact, it would probably be best to stick to one format throughout the files. Up for discussion if it be rgb(a), hsl(a) or something else.

Reply:
@VilppeRiskidev I think you should have 👀  on this one so pinging you. 
On my part should  I fix this or just leave it be?

Reply:
Leave for now, can use whichever, but let's prefer rgba in the future.